### ISSUE TITLE
Fails to parse /> as part of XML body

### ISSUE THREAD

**Issue by jlajoie** [netvl/xml-rs#207]
xml-rs fails to parse XML that contains `/>` as part of it's body. 

Per https://www.w3.org/TR/REC-xml/#syntax
> The ampersand character (&) and the left angle bracket (<) must not appear in their literal form, except when used as markup delimiters, or within a comment, a processing instruction, or a CDATA section. If they are needed elsewhere, they must be escaped using either numeric character references or the strings " &amp; " and " &lt; " respectively. The right angle bracket (>) may be represented using the string " &gt; ", and must, for compatibility, be escaped using either " &gt; " or a character reference when it appears in the string " ]]> " in content, when that string is not marking the end of a CDATA section.

It seems like this should be allowed, as only the `&` and `<` are reserved and _must_ be escaped. `>` is just a may be escaped. As an example in xml-rs, `<b>></b>` is valid XML, just `<b>/></b>` is invalid XML.

Minimally reproducible example:

cargo.toml
```
[package]
name = "xml-minimal-error"
version = "0.1.0"
authors = ["Jeff LaJoie <jeff.lajoie@zenlist.com>"]
edition = "2018"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
xml-rs="0.8"
```

main.rs
```
use xml::reader::ParserConfig;

fn main() {
    let xml_str: &[u8] = b"<b>/></b>";
    let parser_config = ParserConfig::default();
    let parser = parser_config.create_reader(xml_str);

    for event in parser {
        match event {
            Err(e) => {
                println!("Error = {:?}", e);
            }
            _ => {},
        }
    }
}
```

Output
```
➜  xml-minimal-error git:(master) ✗ cargo run
   Compiling xml-minimal-error v0.1.0 (/Users/jlajoie/workspace/xml-minimal-error)
    Finished dev [unoptimized + debuginfo] target(s) in 0.23s
     Running `target/debug/xml-minimal-error`
Error = Error { pos: 1:4, kind: Syntax("Unexpected token: />") }
```

Additionally, this example _does_ work in other language XML parsers. 

NodeJS Example

package.json
```
{
  "name": "xml-minimal",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "fast-xml-parser": "^3.19.0"
  }
}
```

index.js
```
const parser = require('fast-xml-parser');

let jsonObj = parser.parse("<b>/></b>");

console.dir(jsonObj);
```

output
```
➜  node-xml-minimal node index.js
{ b: '/>' }
```

**Comment by bryanburgers** [comment]
I created a PR at #208 (yes, I work with @jlajoie). But it also looks like #195 fixes this.

**Comment by bryanburgers** [comment]
For context, we ran into this issue in a real-world XML document, and the bug caused a production issue for us.

**Comment by eAi** [comment]
We're also seeing this issue downstream in a tool we use - though I wasn't aware that escaping > is optional. This causes XML files that are generated by remodel to fail to be parsed by remodel if they contain > symbols.

https://github.com/rojo-rbx/remodel/issues/58

**Comment by netvl** [comment]
@bryanburgers @eAi @jlajoie this is released in 0.8.4. Please check.

**Comment by netvl** [comment]
Sorry for a very long delay!

**Comment by jlajoie** [comment]
> @bryanburgers @eAi @jlajoie this is released in 0.8.4. Please check.

We're all set with the 0.8.4 release, thanks!

**Comment by netvl** [comment]
Great, so this could be closed, I guess. Thanks!
